x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings.test.tsx

```diff
@@ -47,8 +54,32 @@ describe('RelevanceTuning', () => {
     setMockValues({
       dataLoading: true,
     });
-    const wrapper = shallow(<ResultSettings engineBreadcrumb={['test']} />);
+    const wrapper = subject();
     expect(wrapper.find(ResultSettingsTable).exists()).toBe(false);
     expect(wrapper.find(SampleResponse).exists()).toBe(false);
   });
+
+  it('renders a "save" button that will save the current changes', () => {
+    const buttons = subject().find(EuiPageHeader).prop('rightSideItems') as React.ReactElement[];
```

Comment:
[optional] maybe a candidate for another test helper since it's repeated a few times? e.g. `const buttonsSubject = () => subject().find(EuiPageHeader).prop('rightSideItems') as React.ReactElement[];`

Reply:
3e517553c68984912a6451cecec4228b596bbfc5

I updated tests in Relevance Tuning as well since they were in the same situation.

I took a slightly different approach... I created a selector function (`findButtons`) which accepts a ShallowWrapper (i.e., `subject()`) rather than one that calls `subject()` directly.

It's irrelevant for this instance but as a point of pattern the composability is handy if you have a situation where you need to create the subject differently depending on the test.